### DISCUSSION TITLE
Welcome to InstaTools Discussions!

### DISCUSSION THREAD

**Discussion by new92** [new92/instatools#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KPYahoo** [comment]
@new92 Thanks for this. Is there any documentation somewhere which will help understand use cases which can be implemented using this project?

Thanks

**Comment by new92** [comment]
There isn’t documentation available just yet, but we’re actively working on it and will have it published soon. We’ll be sure to update the repo as soon as it’s ready - stay tuned !

**Comment by KPYahoo** [comment]
@new92 Thanks. I will wait for the documentation to start using this. Looking forward to this eagerly.
